packages/aws-cdk-lib/aws-rds/lib/instance.ts

```diff
@@ -347,6 +347,31 @@ export enum NetworkType {
   DUAL = 'DUAL'
 }
 
+/**
+ * The CA identifier of the DB instance
+ */
+export enum CertificateIdentifier {
```

Comment:
Can you turn the enum into an enum like class instead, with the ability to create a CaCertificate from a custom string: `new CaCertificate('future-identifier')`

```ts
class CaCertificate {
  static public RDS_CA_2019() { return new CaCertificate('rds-ca-2019') }
  public constructor(identifier: string) { }
}
```